mozci/push.py

```diff
@@ -1487,12 +1508,88 @@ def identify_permanent_failures(self, retriggers_count=5, queue_prefix="") -> No
                 )
 
             to_retrigger = retriggers_count - len(retriggers) - scheduled_retriggers
-            if to_retrigger <= 0:
-                # TODO: Analyze the result of the retriggers to classify between backfills and permanent failures
+            if to_retrigger <= 1:
```

Comment:
Why the change from `0` to `1`? This way the retrigger won't get scheduled (line 1516).